### PR TITLE
New play created custom-memes-generator

### PR DESCRIPTION
**Pull request by Amanaakash**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

created a new play called custom-memes-generator 

new play

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# 
image

video preview : https://youtu.be/EsAuhMLLhQ0

### CODE REVIEW

File: src/plays/custom-memes-generator/CustomMemesGenerator.css [hunk 1]
```diff
@@ -0,0 +1,185 @@
+.cmg-body {
```
Comment (atapas) [inline src/plays/custom-memes-generator/CustomMemesGenerator.css:1]: @Amanaakash Why don't you use this class `cmg-meme-main` as prefix for all the styles?

**Review by atapas — CHANGES_REQUESTED** [review]

**Review by Shivam-Katare — CHANGES_REQUESTED** [review]

File: src/plays/custom-memes-generator/CustomMemesGenerator.css (hunk 1 above)
Comment (Amanaakash) [reply src/plays/custom-memes-generator/CustomMemesGenerator.css:1]: i changed the styles to cmg-meme-main as their prefix

File: src/plays/custom-memes-generator/Meme.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+          />
+        </div>
+
+        <button className="main-form-button" onClick={getNewUrl}>
+          Generate a new meme image 🖼
+        </button>
+      </div>
+
+      <div className="main-meme-image-container">
+        <img className="main-meme-image" src={meme.randomImage} />
```
[the remark this answers is not in the document]
Comment (Amanaakash) [reply src/plays/custom-memes-generator/Meme.jsx]: alt attribute for the image added!

File: src/plays/custom-memes-generator/Readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Custom Memes Generator
+
+It is a react app which generates meme templates and lets you write over it to make a custom meme
+
+## Play Demographic
+
+- Language: js
```
[the remark this answers is not in the document]
Comment (Amanaakash) [reply src/plays/custom-memes-generator/Readme.md]: changed to JavaScript

**Review by Amanaakash — COMMENTED** [review]
The requested changes have been addressed please review

File: src/plays/custom-memes-generator/Readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+- User: amanaakash
+- Gihub Link: https://github.com/amanaakash
+- Blog:
+- Video:
+
+## Implementation Details
+
+Update your implementation idea and details here
+
+## Consideration
```
[the remark this answers is not in the document]
Comment (Amanaakash) [reply src/plays/custom-memes-generator/Readme.md]: removed it !

**Review by Shivam-Katare — DISMISSED** [review]
LGTM!

**Comment by Amanaakash** [comment]
having some issues with this pr ill close it and open a new one after testing

**Pull request closed without merging**